arraycontext/impl/pyopencl/taggable_cl_array.py

```diff
@@ -119,9 +119,9 @@ def with_tagged_axis(self, iaxis: int,
         """
         Returns a copy of *self* with *iaxis*-th axis tagged with *tags*.
         """
-        new_axes = (self.axes[:iaxis]
-                    + (self.axes[iaxis].tagged(tags),)
-                    + self.axes[iaxis+1:])
+        new_axes = ((*self.axes[:iaxis],
+                     self.axes[iaxis].tagged(tags),
+                     *self.axes[iaxis + 1:]))
```

Comment:
```suggestion
        new_axes = (*self.axes[:iaxis],
                    self.axes[iaxis].tagged(tags),
                    *self.axes[iaxis + 1:])
```
? Yeah, I think the main selling point of this is that it works with any iterable, which is not a big plus here..